### ISSUE TITLE
VxMark: make rate change audio feedback similar to how volume feedback works 

### ISSUE THREAD

**Issue by carolinemodic** [votingworks/vxsuite#7717]
Rate changes don't seem to have any audio feedback in mark/markscan. We should make it work like the volume changes where it only reads out if nothing else is being said but does provide some audio cue in that case. 

Original Issue:
Not sure if this is an issue but we only run xmodmap properly on boot so if the ATI is plugged in after boot the volume up/down and rate change buttons do not work properly. The sound level is also different. When I was testing on the qa image I made 12-18-2025 at the atrium the rate change buttons did not work but the volume change buttons do. On my dev laptop it seems like the keys are mapping to what we expect so I'm not sure what the problem is in the QA image. The volume buttons DO work.

**Comment by arsalansufi** [comment]
This one _may_ be explained by using out of date election packages missing the audio cues for the rate change buttons
